`X2jOptions` is exported twice, making TS build fail

* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?
* [ ] Have you checked the [docs](https://github.com/NaturalIntelligence/fast-xml-parser/tree/master/docs) for helpful APIs and examples?

### Description
<!-- Please provide following data to avoid long communication and fast resolution. --> 

The `X2jOptions` type is both exported [here](https://github.com/NaturalIntelligence/fast-xml-parser/blob/c20fbd675dcb45a5e012115b1d5fedbb2eaf2577/lib/fxp.d.cts#L57) (at declaration site) and [here](https://github.com/NaturalIntelligence/fast-xml-parser/blob/c20fbd675dcb45a5e012115b1d5fedbb2eaf2577/lib/fxp.d.cts#L518) (in `fxp` namespace).

### Input
<!-- Please include your sample code or steps to reproduce here -->

With `typescript@5.1.6`, build a project with the following import:

### Code
```js
import {XMLParser, XMLValidator} from 'fast-xml-parser'
```

### Output
<!-- Please include output data or error here -->

Build error:
```
node_modules/fast-xml-parser/lib/fxp.d.cts:527:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.

527 export = fxp;
    ~~~~~~~~~~~~~
```

### Expected data
<!-- Please include expected output data here -->

Successful build.

**Would you like to work on this issue?**
<!-- choose one by changing [ ] to [x] -->
- [x] Yes
- [ ] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates. Visit [SoloThought](https://solothought.com) to know about recent features.

<!-- Love Fast-XML-Parser? Please consider supporting us:
👉   https://opencollective.com/fast-xml-parser/donate
👉   https://paypal.me/naturalintelligence 
👉👉  https://github.com/sponsors/NaturalIntelligence -->

--- a/lib/fxp.d.cts
+++ b/lib/fxp.d.cts
@@ -54,7 +54,7 @@ type ProcessEntitiesOptions = {
   tagFilter?: ((tagName: string, jPath: string) => boolean) | null;
 };
 
-export type X2jOptions = {
+type X2jOptions = {
   /**
    * Preserve the order of tags in resulting JS object
    * 

